### PR TITLE
Add tool parameter argument to parameter help

### PR DESCRIPTION
**Pull request by guerler**
This PR automatically adds the tool parameter argument name  to the parameters help text.
See: https://github.com/galaxyproject/galaxy/pull/8

### CODE REVIEW

**Comment by hexylena** [comment]
cc @bgruening

**Comment by bgruening** [comment]
Great @guerler - Thanks for implementing! :+1:

File: client/galaxy/scripts/mvc/form/form-input.js [hunk 1]
```diff
@@ -124,6 +124,9 @@ define([], function() {
             tmp +=              '<div class="ui-table-form-info">';
             if (options.help) {
                 tmp +=              options.help;
+                if (options.argument) {
```
Comment (nsoranzo) [inline client/galaxy/scripts/mvc/form/form-input.js]: Maybe here we should check that `options.help` does not already contain `options.argument` to make it possible for tool developers to cleanly support old and new Galaxy syntax.

Comment (hexylena) [inline client/galaxy/scripts/mvc/form/form-input.js]: yep, there are a ton of IUC tools which encode the argument like this.

**Comment by nsoranzo** [comment]
:+1:

**Comment by jmchilton** [comment]
:+1:

**Comment by yhoogstrate** [comment]
@erasche @guerler @nsoranzo 
 `<param name="max" label="Take the max" argument="--max" />` will show no help section and thus won't show the `(--max)` argument.
Is it on purpose that if `help=` is not being set, it won't be created by setting `argument=`, or is this a small bug?

**Comment by nsoranzo** [comment]
I'd say small bug.

**Comment by bgruening** [comment]
I guess this is a bug. If you include an empty help it will be shown.
On Feb 2, 2016 1:34 PM, yhoogstrate notifications@github.com wrote:@erasche @guerler @nsoranzo 
 <param name="max" label="Take the max" argument="--max" /> will show no help section and thus won't show the (--max) argument.
Is it on purpose that if help= is not being set, it won't be created by setting argument=, or is this a small bug?

**Pull request merged**